tests/test_graph_types.py

```diff
@@ -113,14 +114,52 @@ def node_to_create(required_dep: int, optional_dep: int = 1) -> str:
     }
 
 
-def test_create_hamilton_node_missing_version():
+def test_create_hamilton_config_node_version():
+    """Config nodes now return the name as the version."""
     n = Node("foo", int, node_source=NodeType.EXTERNAL)
     hamilton_node = graph_types.HamiltonNode.from_node(n)
     # the above will have no specified versions
+    assert hamilton_node.version == "foo"
+    assert hamilton_node.as_dict()["version"] == "foo"
+
+
+def test_create_hamilton_node_missing_version():
+    """We contrive a case where originating_functions is None."""
+
+    def foo(i: int) -> int:
+        return i
+
+    n = Node("foo", int, "", foo, node_source=NodeType.STANDARD)
+    hamilton_node = graph_types.HamiltonNode.from_node(n)
+    # the above will have no specified versions
     assert hamilton_node.version is None
     assert hamilton_node.as_dict()["version"] is None
 
 
+def test_hamilton_graph_version_normal():
+    dr = driver.Builder().with_modules(no_parallel).build()
+    graph = graph_types.HamiltonGraph.from_graph(dr.graph)
+    # assumption is for python 3
+    if sys.version_info.minor == 8:
+        hash_value = "0a375f3366590453dea8927d4c02c15dc090f8be42e9129d9a1139284eac920c"
```

Comment:
Huh? This is a bit confusing... Why is it these hashes/why is it different between versions? Best to just test its a certain amount of characters IMO

Reply:
yeah hashing can change between python versions. Only found it to be different between 3.8 and the rest. Since I've already gone and done it, no need to shorten.